src/pages/brain/page-details.scss

```diff
@@ -0,0 +1,33 @@
+@import "react-notion-x/src/styles.css";
+:root {
+  --fg-color: var(--chakra-colors-chakra-body-text) !important;
+  --fg-color-2: var(--chakra-colors-chakra-body-text) !important;
+  --fg-color-3: var(--chakra-colors-chakra-body-text) !important;
+  --fg-color-5: var(--chakra-colors-gray-600) !important;
+}
+
+.chakra-link {
```

Comment:
this would go into the component's theme config file (link.ts), but we don't wan't to override it like this globally so just inline it where wanted